jupiterone/resource_dashboard_test.go

```diff
@@ -27,26 +30,26 @@ func TestDashboard_Basic(t *testing.T) {
 	resource.Test(t, resource.TestCase{
 		PreCheck:                 func() { testAccPreCheck(t) },
 		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories(recordingClient),
-		CheckDestroy:             testAccCheckDashboardDestroy(ctx, directClient),
+		CheckDestroy:             testAccCheckDashboardDestroy(ctx, testDashboardResourceName, directClient),
```

Comment:
There was a bug in the destroy test logic for this acceptance test that prevented successful runs - this is now fixed. This test was modeled after the resource_rule_test which uses a similar testing style that I found to be easier to reason about with passing exact ids and not allowing the testAccCheckDashboardDestroy to grab the resource id from the state/context.